components/developer/jenkins/Makefile

```diff
@@ -14,76 +14,36 @@
 #
 include ../../../make-rules/shared-macros.mk
 
-# NOTE: Before bumping to the new core version, take a look at
-#   https://jenkins.io/changelog/ (weekly)
-#   https://jenkins.io/changelog-stable/ (LTS)
-# for "weather report" (user votes on good/bad state of the release).
-# At some point it may make sense to deliver packaged LTS releases,
-# and the initial recipe supports that notion already just in case,
-# but no LTS issued so far includes updated libzfs.jar (v0.8+ needed).
-# Note that for LTS it would make sense to define indexed "additional
-# source" variables below, see e.g. docbook for inspiration.
+# NOTE: Initial jenkins recipe had one component for both the JAR
+# file downloads and the OpenIndiana integration helper packages.
+# Due to this, initial version for helper packages starts from that.
 
 COMPONENT_NAME=		jenkins
-COMPONENT_MAJOR_VERSION=	2
-COMPONENT_MINOR_VERSION=	65
-COMPONENT_VERSION=	$(COMPONENT_MAJOR_VERSION).$(COMPONENT_MINOR_VERSION)
-COMPONENT_ARCHIVE=	$(COMPONENT_NAME)-$(COMPONENT_VERSION).war
-COMPONENT_FMRI=		developer/jenkins-core-weekly
-COMPONENT_CLASSIFICATION=	Development/Java
-COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_VERSION=	2.65
```

Comment:
No, preferably it should restart from 1.0 but I guess it is too late :)
This is in fact version of the OS-integration bits like jenkins-common and jenkins-standalone, which change on a much different schedule than the weekly.